File: src/main/java/com/haozileung/web/service/permission/IRoleService.java
package com.haozileung.web.service.permission;


import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.Role;


public interface IRoleService {

    void save(Role dictionary);

    void update(Role dictionary);

    void delete(Role dictionary);

    PageResult<Role> query(Role dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IRoleService.kt
package com.haozileung.web.service.permission


import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.permission.Role


interface IRoleService {

    fun save(dictionary: Role)

    fun update(dictionary: Role)

    fun delete(dictionary: Role)

    fun query(dictionary: Role, pageNo: Int, pageSize: Int): PageResult<Role>
}
